Makefile

```
@@ -124,6 +124,9 @@ clean:
 distclean: clean
 	rm -rf vendor/
 
+cl:
```

However, this should depend on `cmd/sdkch/main.go`
```suggestion
cl: cmd/sdkch/main.go
```